doc/src/manual/documentation.md

```diff
@@ -585,6 +585,22 @@ parentheses, `( )`, is the URL.
 A paragraph containing a link to [Julia](http://www.julialang.org).
 ```
 
+It's also possible to add cross-references to other documented functions/methdods/variables within
+the Julia documentation itself. For example:
+
+```julia
+"""
+    eigvals!(A,[irange,][vl,][vu]) -> values
+
+Same as [`eigvals`](@ref), but saves space by overwriting the input `A`, instead of creating a copy.
+"""
+```
+
+This will create a link in the generated docs to the `eigvals` documentation
+(which has more information about what this function actually does). It's good to include
+cross references to mutating/non-mutating versions of a function, or to highlight a difference
+between two similar-seeming functions.
```

Comment:
Should make it clear here that `@ref` is not markdown syntax, and requires use of Documenter. (Might better to mention this kind of thing in CONTRIBUTING.md instead?)